### PR TITLE
Build the sitemap from the release tags

### LINKED ISSUE #971: Use release tags to determine what is included in sitemap
* Change PurlResource.crawlable? to use "PURL sitemap" release tag from public XML to determine what is added to sitemap.
* Remove the existing setting.
* Run test sitemap generation to make sure it is working properly.

### PR DESCRIPTION
**Pull request by jcoyne**
Fixes #971

### CODE REVIEW

File: app/models/purl_resource.rb [hunk 1]
```diff
@@ -70,9 +80,9 @@ def containing_purl_collections
 
   # Can be crawled / indexed by a crawler, e.g. Googlebot
   def crawlable?
```
Comment (justinlittman) [inline app/models/purl_resource.rb:82]: Can you get rid of this method?

**Review by justinlittman — CHANGES_REQUESTED** [review]

File: app/models/purl_resource.rb (hunk 1 above)
Comment (jcoyne) [reply app/models/purl_resource.rb:82]: It's called in `app/views/purl/show.html.erb`

Comment (justinlittman) [reply app/models/purl_resource.rb:82]: Ah, OK.

**Review by justinlittman — APPROVED** [review]

**Comment by justinlittman** [comment]
This has been tested so merging.

**Pull request merged**